lib/src/Base/Type/Point.cxx

```diff
@@ -92,6 +92,16 @@ Point Point::clean(const Scalar threshold) const
   return result;
 }
 
+/* Check if the components are in increasing or decreasing order */
+Bool Point::isMonotonic() const
```

Comment:
Sounds weird to me, this is not the mathematical definition.

Reply:
@dbarbier Do you have a better name? I was thinking about the value of the component as a function of its index when I chose this name. If I adopt a more geometric view, then I check if the point is in a particular cone but I can't figure out how to name the method according to this point of view...

Reply:
@regislebrun Sorry if I was unclear; my point is that a monotonic function may stall, you should replace < binary operator by <= operator, but this is not what you want. A French/English translation would give isStrictlyMonotonic, but it does not seem to be widely used.  Anyway I do not find a better name.

Reply:
I think the name is fine, but maybe the docstring could mention "strictly increasing".

Reply:
@dbarbier @jschueller I added 'strictly' to the description of the method. Some times ago, we had a hard time with a reviewer concerning the difference between increasing, non-decreasing and even strictly increasing. For the reviewer the first one was for x_1<x2, the second one for x_1<=x_2 and the last one was a repetition. The paper was rejected, we reworked it a lot and resubmitted it to the exact same review. We probably got new reviewers, as this time strictly increasing was accepted without comment!

Reply:
Looks like Wikipedia & wolfram definitions agree with Denis:
https://en.wikipedia.org/wiki/Monotonic_function
http://mathworld.wolfram.com/MonotonicFunction.html